pytorch_lightning/tuner/batch_size_scaling.py

```diff
@@ -25,6 +25,7 @@
 
 
 def scale_batch_size(trainer,
+                     fs,
```

Comment:
why pass `fs` into the function instead of calling `get_filesystem` based on `save_path`?

Reply:
why didn't I think of that.. updated thanks :)